spec/cli/wordlist_option_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+require 'ronin/wordlists/cli/wordlist_option'
+require 'ronin/wordlists/cli/command'
+
+RSpec.describe Ronin::Wordlists::CLI::WordlistOption do
+  class WordlistOptionCommandMock < Ronin::Wordlists::CLI::Command
+    include Ronin::Wordlists::CLI::WordlistOption
+  end
+
+  describe '.included' do
+    subject { WordlistOptionCommandMock }
```

Comment:
Define a `let(:command_class) { ... }` to avoid having to re-type the class name in multiple places, assuming we have to add more specs for `WordlistOption`.